Library/include/CSP/Multiplayer/SpaceEntitySystem.h

```diff
@@ -320,6 +325,10 @@ class CSP_API SpaceEntitySystem
 	using SpaceEntityList = csp::common::List<SpaceEntity*>;
 
 	SpaceEntityList Entities;
+	SpaceEntityList Avatars;
```

Comment:
Are these being made public for the multiplayertests? Can that test class not be made a friend rather?

Reply:
Ah right, missed that they were protected.